liquibase.yaml

```
@@ -0,0 +1,181 @@
+package:
+  name: liquibase
+  version: "4.31.1"
+  epoch: 0
+  description: "Liquibase is a database schema change management solution that enables you to revise and release database changes faster and safer from development to production."
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - liquibase-package-manager
+      - openjdk-${{vars.java-version}}-default-jvm
+
+vars:
+  java-version: '17'
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - maven
+      - openjdk-${{vars.java-version}}-default-jdk
+      - wolfi-base
+  environment:
+    LANG: en_US.UTF-8
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/liquibase/liquibase
+      tag: v${{package.version}}
+      expected-commit: 12ab20b176e5659622c240b4ba4f056ccf17c27d
+
+  - name: Setup
+    runs: |
+      # Reversion liquibase.build.properties
+      # REF: https://github.com/liquibase/liquibase/blob/master/.github/util/re-version.sh#L87
+      sed -i -e "s/build.version=.*/build.version=${{package.version}}/" liquibase-standard/src/main/resources/liquibase.build.properties
+
+  - name: Build
+    runs: |
+      # Build liquibase
+      ./mvnw -B clean package -DskipTests=true -P !liquibase-commercial
+
+  - name: Install
+    runs: |
+      # Setup destination dir
+      mkdir -p ${{targets.destdir}}/opt/java/openjdk
```

Overwritten by the symlink below

```suggestion
      mkdir -p ${{targets.destdir}}/opt/java
```